task/jira-get-ticket-status/0.1/jira-get-ticket-status.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: jira-get-ticket-status
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/categories: Automation
+    tekton.dev/tags: jira, get, ticket, status
+    tekton.dev/displayName: "Jira - Get Ticket Status"
+    tekton.dev/platforms: "linux/amd64"
+spec:
+  description: >-
+    This task helps you to get the current status of a Jira ticket.
+  params:
+    - name: ticket_id
+      description: Jira Ticket ID
+      type: string
+    - name: ssl_verify
+      default: "true"
+      description: Verify or not SSL certificates
+  results:
+    - name: ticket_status
+      description: Jira Ticket status
+  stepTemplate:
+    envFrom:
+      - secretRef:
+          name: jira-env-secret
```

Comment:
can we try making this secret name as parameterized?

Reply:
yes! done